index.html

```diff
@@ -173,6 +177,26 @@ <h4>Suitable PR content</h4>
 mailing list. Make sure to include the URL that you want on w3id.org, the
 URL that you want to redirect to, and the HTTP code that you want to use
 when redirecting. An administrator will then create the redirect for you.
+        </p>
+        <h4>Link checking</h4>
+        <p>
+A simple <a herf="https://travis-ci.org/perma-id/w3id.org">Travis-CI</a> job
+(see <a href="https://github.com/perma-id/w3id.org/blob/master/.travis.yml">.travis.yml</a>) will extract all https://w3id.org/
+URIs from <code>*/README.md</code> and check them with
+<code><a href="https://wummel.github.io/linkchecker/">linkchecker</a></code>.
+In theory, this will catch two kinds of errors:
```

Comment:
```suggestion
(see
<a href="https://github.com/perma-id/w3id.org/blob/master/.travis.yml">.travis.yml</a>)
will extract all <code>https://w3id.org/</code> URIs from
<code>*/README.md</code> and check them with
<code><a href="https://wummel.github.io/linkchecker/">linkchecker</a></code>.
In theory, this will catch two kinds of errors:
```